patches/minecraft/net/minecraft/server/management/PlayerList.java.patch

```diff
@@ -146,14 +146,20 @@
          return entityplayermp;
      }
  
-@@ -556,15 +601,20 @@
+@@ -556,15 +601,26 @@
  
      public void func_187242_a(EntityPlayerMP p_187242_1_, int p_187242_2_)
      {
 +        transferPlayerToDimension(p_187242_1_, p_187242_2_, field_72400_f.func_71218_a(p_187242_2_).func_85176_s());
 +    }
 +
++    // TODO: Remove (1.13)
 +    public void transferPlayerToDimension(EntityPlayerMP p_187242_1_, int p_187242_2_, net.minecraft.world.Teleporter teleporter)
```

Comment:
You should deprecate this first

Reply:
It's not needed to deprecate these methods. The change is source-compatible, a recompile is all thats needed.